hashicorp/boundary-ui
# update breadcrumbs to use hds::breadcrumbs

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-9424)

## Description


Add `@bagaar/ember-breadcrumbs`
Add component that uses HDS::Breadcrumbs
Remove previous custom breadcrumbs
Remove controller files that were only being used for breadcrumbs
Remove unnecessary unit test files
Clean-up any tests

## How to Test


1. Go to routes and verify correct breadcrumbs are displayed.


:technologist: [Admin preview](PATH_TO_FEATURE)

## Checklist:
